### ISSUE TITLE
Unable to rip twitter urls since mid-January

### ISSUE THREAD

**Issue by dprski33** [rg3/youtube-dl#12052]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
- Use *Preview* tab to see how your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2017.02.10*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [ ] I've **verified** and **I assure** that I'm running youtube-dl **2017.02.10**
2017.02.07
### Before submitting an *issue* make sure you have:
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

```
youtube-dl -v -k -f mp4 -o '/tmp/twitter682662065862012928.mp4'  'https://twitter.com/paigekrill/status/682662065862012928'
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'-k', u'-f', u'mp4', u'-o', u'/tmp/twitter682662065862012928.mp4', u'https://twitter.com/paigekrill/status/682662065862012928']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2017.02.07
[debug] Python version 2.7.6 - Linux-3.13.0-93-generic-x86_64-with-Ubuntu-14.04-trusty
[debug] exe versions: ffmpeg N-80901-gfebc862, ffprobe N-80901-gfebc862
[debug] Proxy map: {}
[twitter] 682662065862012928: Downloading webpage
[twitter:card] 682662065862012928: Downloading webpage
[twitter:card] 682662065862012928: Downloading m3u8 information
[debug] Invoking downloader on u'https://video.twimg.com/ext_tw_video/682661265647378432/pu/pl/720x1280/AEgA2Fdd0Ndma0ts.m3u8'
[download] Destination: /tmp/twitter682662065862012928.mp4
[debug] ffmpeg command line: ffmpeg -y -headers 'Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.7
Accept-Language: en-us,en;q=0.5
Accept-Encoding: gzip, deflate
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:10.0) Gecko/20150101 Firefox/47.0 (Chrome)
' -i https://video.twimg.com/ext_tw_video/682661265647378432/pu/pl/720x1280/AEgA2Fdd0Ndma0ts.m3u8 -c copy -f mp4 file:/tmp/twitter682662065862012928.mp4.part
ffmpeg version N-80901-gfebc862 Copyright (c) 2000-2016 the FFmpeg developers
  built with gcc 4.8 (Ubuntu 4.8.4-2ubuntu1~14.04.3)
  configuration: --extra-libs=-ldl --prefix=/opt/ffmpeg --mandir=/usr/share/man --enable-avresample --disable-debug --enable-nonfree --enable-gpl --enable-version3 --enable-libopencore-amrnb --enable-libopencore-amrwb --disable-decoder=amrnb --disable-decoder=amrwb --enable-libpulse --enable-libfreetype --enable-gnutls --enable-libx264 --enable-libx265 --enable-libfdk-aac --enable-libvorbis --enable-libmp3lame --enable-libopus --enable-libvpx --enable-libspeex --enable-libass --enable-avisynth --enable-libsoxr --enable-libxvid --enable-libvidstab
  libavutil      55. 28.100 / 55. 28.100
  libavcodec     57. 48.101 / 57. 48.101
  libavformat    57. 41.100 / 57. 41.100
  libavdevice    57.  0.102 / 57.  0.102
  libavfilter     6. 47.100 /  6. 47.100
  libavresample   3.  0.  0 /  3.  0.  0
  libswscale      4.  1.100 /  4.  1.100
  libswresample   2.  1.100 /  2.  1.100
  libpostproc    54.  0.100 / 54.  0.100
Input #0, hls,applehttp, from 'https://video.twimg.com/ext_tw_video/682661265647378432/pu/pl/720x1280/AEgA2Fdd0Ndma0ts.m3u8':
  Duration: 00:00:07.44, start: 0.000000, bitrate: 0 kb/s
  Program 0 
    Metadata:
      variant_bitrate : 0
    Stream #0:0: Video: h264 (Main) ([27][0][0][0] / 0x001B), yuv420p, 720x1280, 29.97 fps, 29.98 tbr, 90k tbn, 59.94 tbc
    Stream #0:1: Audio: aac (LC) ([15][0][0][0] / 0x000F), 44100 Hz, mono, fltp, 125 kb/s
[mp4 @ 0x286f540] Using AVStream.codec to pass codec parameters to muxers is deprecated, use AVStream.codecpar instead.
    Last message repeated 1 times
Output #0, mp4, to 'file:/tmp/twitter682662065862012928.mp4.part':
  Metadata:
    encoder         : Lavf57.41.100
    Stream #0:0: Video: h264 ([33][0][0][0] / 0x0021), yuv420p, 720x1280, q=2-31, 29.97 fps, 29.98 tbr, 90k tbn, 90k tbc
    Stream #0:1: Audio: aac (LC) ([64][0][0][0] / 0x0040), 44100 Hz, mono, 125 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
  Stream #0:1 -> #0:1 (copy)
Press [q] to stop, [?] for help
[mp4 @ 0x286f540] Malformed AAC bitstream detected: use the audio bitstream filter 'aac_adtstoasc' to fix it ('-bsf:a aac_adtstoasc' option with ffmpeg)
av_interleaved_write_frame(): Operation not permitted
[mp4 @ 0x286f540] Malformed AAC bitstream detected: use the audio bitstream filter 'aac_adtstoasc' to fix it ('-bsf:a aac_adtstoasc' option with ffmpeg)
Error writing trailer of file:/tmp/twitter682662065862012928.mp4.part: Operation not permittedframe=    2 fps=0.0 q=-1.0 Lsize=      71kB time=00:00:00.11 bitrate=4975.9kbits/s speed= 168x    
video:72kB audio:2kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
Conversion failed!


ERROR: ffmpeg exited with code 1
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/bin/youtube-dl/__main__.py", line 19, in <module>
    youtube_dl.main()
  File "/usr/bin/youtube-dl/youtube_dl/__init__.py", line 452, in main
    _real_main(argv)
  File "/usr/bin/youtube-dl/youtube_dl/__init__.py", line 442, in _real_main
    retcode = ydl.download(all_urls)
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1796, in download
    url, force_generic_extractor=self.params.get('force_generic_extractor', False))
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 707, in extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 778, in process_ie_result
    new_result, download=download, extra_info=extra_info)
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 753, in process_ie_result
    return self.process_video_result(ie_result, download=download)
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1440, in process_video_result
    self.process_info(new_info)
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1704, in process_info
    success = dl(filename, info_dict)
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1646, in dl
    return fd.download(name, info)
  File "/usr/bin/youtube-dl/youtube_dl/downloader/common.py", line 353, in download
    return self.real_download(filename, info_dict)
  File "/usr/bin/youtube-dl/youtube_dl/downloader/external.py", line 44, in real_download
    self.get_basename(), retval))
  File "/usr/bin/youtube-dl/youtube_dl/downloader/common.py", line 162, in report_error
    self.ydl.report_error(*args, **kargs)
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 570, in report_error
    self.trouble(error_message, tb)
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 532, in trouble
    tb_data = traceback.format_list(traceback.extract_stack())
```
---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: https://twitter.com/stoolive/status/800424897881186304
- Single video: https://twitter.com/CasadyMarie/status/825917927526453248
- Single video: https://twitter.com/paigekrill/status/682662065862012928

Note that **youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/rg3/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free)**. In order for site support request to be accepted all provided example URLs should not violate any copyrights.

---

### Description of your *issue*, suggested solution and other information
I was able to rip twitter urls in early January, but no longer.

Example twitter urls that fail to rip:
https://twitter.com/stoolive/status/800424897881186304
https://twitter.com/CasadyMarie/status/825917927526453248
https://twitter.com/paigekrill/status/682662065862012928

**Comment by ThomasChr** [comment]
I have no problem with twitter urls.
I find your ffmpeg version rather odd (N-80901-gfebc862) - please try another one.

**Comment by remitamine** [comment]
already fixed in 2017.02.10.
